iroh/src/node/builder.rs

```diff
@@ -56,6 +53,11 @@ const DEFAULT_GC_INTERVAL: Duration = Duration::from_secs(60 * 5);
 const MAX_CONNECTIONS: u32 = 1024;
 const MAX_STREAMS: u64 = 10;
 
+type BoxedServerEndpoint = quic_rpc::transport::boxed::ServerEndpoint<
```

Comment:
Maybe it should be ServiceEndpoint<S> instead of ServerEndpoint<In, Out>. But that would require a new version of quic-rpc, so not today...

Reply:
didn't @Frando PR that at some point?

Reply:
We got a ton of places in quic-rpc where we switched from Channel<In, Out> to ServiceChannel<S>, but not all places.